src/Microsoft.AspNetCore.Http.Connections.Client/Internal/SendUtils.cs

```diff
@@ -15,7 +15,7 @@ namespace Microsoft.AspNetCore.Http.Connections.Client.Internal
 {
     internal static class SendUtils
     {
-        public static async Task SendMessages(Uri sendUrl, IDuplexPipe application, HttpClient httpClient, ILogger logger)
+        public static async Task SendMessages(Uri sendUrl, IDuplexPipe application, HttpClient httpClient, ILogger logger, CancellationToken cancellationToken)
```

Comment:
You can use default here to make this optional and avoid passing in `CancellationToken.None`